pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 testdox_format = plaintext
-addopts = --testdox --timeout 20
+addopts = --testdox --timeout 200
```

Comment:
>200 [](start = 30, length = 3)

I will change this before merge

Reply:
:thumbsup: 

---
In reply to: [330807274](https://github.com/Azure/azure-iot-sdk-python/pull/340#discussion_r330807274) [](ancestors = 330807274)